### PR TITLE
Ray's Statements Changes

### PR DESCRIPTION
**Pull request by teryhill**
Credit to Ray Magauran for this work. New statement format using html or
the standard PDF . #315

### CODE REVIEW

**Comment by teryhill** [comment]
@tony @aethelwulffe  any interest in this

**Comment by aethelwulffe** [comment]
Yes, absolutely. 
  While this reminds me that we still have crap named sl_, this is a good piece.

**Comment by teryhill** [comment]
What else? I have a WIP to ditch the sl

**Comment by aethelwulffe** [comment]
Yeah, that's where I am headed with that...that's all...

File: interface/billing/sl_eob_search.php [hunk 1]
```diff
@@ -33,6 +33,14 @@
 require_once("$srcdir/sl_eob.inc.php");
 require_once("$srcdir/formatting.inc.php");
 require_once("$srcdir/classes/class.ezpdf.php");//for the purpose of pdf creation
+require_once("$srcdir/api.inc");
+require_once("$srcdir/forms.inc");
+require_once("$srcdir/../controllers/C_Document.class.php");
```
Comment (tmccormi) [inline interface/billing/sl_eob_search.php]: why were all these included?  I see nothing in this code that used document class etc ...

**Comment by teryhill** [comment]
removed them. I am not sure why they are there but I removed them and it did not cause a problem

**Comment by aethelwulffe** [comment]
Thank you very much.
That was a good part of what I was hoping to see.

**Comment by teryhill** [comment]
The portal will be able to send an invoice to the patient as a message and a document. Need to get the asset stuff approved and in place

**Comment by aethelwulffe** [comment]
Hopefully we will move forward some pretty quick here in the next day or two...

**Comment by aethelwulffe** [comment]
I think you took care of Tony's concerns.  Not seeing his thumb on this, but he is incommunicado and I hate to see good stuff go stale.  I will merge this and deal with getting yelled at later.

**Pull request merged**